packages/viewer/src/components/viewer/viewer.tsx

```diff
@@ -230,6 +230,19 @@ export class Viewer {
   @Prop({ attribute: null })
   public featureHighlighting?: FeatureHighlightOptions;
 
+  /**
+   * Specifies when a feature map is returned from rendering. Feature maps
+   * include information about the surfaces, edges and cross sections that are
+   * in a frame.
+   *
+   * Possible values are:
+   *
+   * * `undefined`: A feature map is never requested.
+   * * `final`: A feature map is only requested on the final frame.
+   * * `all`: A feature map is requested for every frame.
+   */
+  @Prop() public featureMaps?: FrameType;
```

Comment:
Any value in adding a similar comment here to depth buffers? E.g. `Feature maps can increase the amount of data that's sent to a client and can impact rendering performance. Values of "undefined" or "final
 should be used when needing the highest rendering performance.`